### ISSUE TITLE
Broken search 

### ISSUE THREAD

**Issue by elmcip** [elmcip/elmcip#471]
The issue arrives after you search a term and then try to Filter by 
Content Type.

So, let's say you search "game".

It takes you to this page: https://elmcip.net/search/game

But then when you click the Creative Work filter box, you get this url: 
https://elmcip.net/?f%5B0%5D=type%3Awork

It does this for any search term.

So it's not actually filtering the results. Instead it's just pushing 
you back to the main page, more or less.

**Comment by hannahackermans** [comment]
Yes, I was just contacted by a student about this error. I told them to use google search (ie "elmcip.net: game") while we are fixing this. 

Is this connected to the NIRD maintenance or is it an error of our own?

**Comment by steinmb** [comment]
Might be related to the changes NIRD did last week. Anyway, I have notified NIRD support, switch our search backend to our backup solution and rebuilt the search index. Search should in the mean time work as expected. 

As soon as the https://solr.apache.org/ is stable again I'll switch back the search back end.

**Comment by steinmb** [comment]
I see... we have two issues in here....

**Comment by steinmb** [comment]
We rolled this change in august to the search api. It might have broken our search...

commit a682f6c7d7ad62d53764d44c478aa763677bdbb2
Author: Stein Magne Bjorklund <steinmb@smbjorklund.com>
Date:   Tue Aug 24 10:19:35 2021 +0200

    Search API (search_api)  7.x-1.27 -> 7.x-1.28

    https://www.drupal.org/project/search_api/releases/7.x-1.28

**Comment by steinmb** [comment]
Yes, the original issue reported by @elmcip is prob from https://www.drupal.org/node/2159827 - Seems to have broken the search for other to.

**Comment by steinmb** [comment]
Posted to the Drupal issue:
https://www.drupal.org/project/search_api/issues/2159827#comment-14237030

**Comment by steinmb** [comment]
Rolled a change to test

commit 4c9c5b25cf521da384fe8c307e09c4fdc94e3340 (HEAD -> master, origin/master, origin/HEAD)
Author: Stein Magne Bjorklund <steinmb@smbjorklund.com>
Date:   Mon Sep 27 15:09:42 2021 +0200

    Search API 2159827 - base path missing

        Rolling 2159827 #35 patch to make facets work.

**Comment by steinmb** [comment]
Postponed until there is a official fix. The patch in production works for us but not for everybody: https://www.drupal.org/project/search_api/issues/2159827

**Comment by elmcip** [comment]
Huh. Was this ever fixed in production? Faceted search is having problems. For example, Caitlin Fisher has a lot of works in creative works that are not turning up with the creative work facet checked. So if this was fixed it seems to be back.
